Userland/Applications/Piano/RollWidget.cpp

```diff
@@ -88,6 +89,63 @@ void RollWidget::paint_event(GUI::PaintEvent& event)
     int horizontal_notes_to_paint = horizontal_paint_area / m_note_width;
 
     GUI::Painter painter(*this);
+
+    // Draw the background, if necessary.
+    if (viewport_changed() || paint_area != m_background->height()) {
+        dbgln("background redraw");
```

Comment:
This `dbgln` gets added and then removed in the next commit

Reply:
Yes, this is an issue with the previous commit that awaits merge. I think it's OK to leave it in if both PRs are merged successively.